docs/community/setup.md

```diff
@@ -178,8 +178,6 @@ pass.
 To run the tests with `nox`, run the following commands:
 
 ```console
-$ nox -s compile
```

Comment:
So... when I was working on this on my computer, I thought that in order for me to successfully run `nox -s test` (without making any changes to `noxfile.py`) I had to first run `compile` AND `docs`, like so:

```console
nox -s compile
nox -s docs
nox -s test
```

But then, wanting to double-check my assumptions, I started from a clean, new repo and discovered that I only had to run `docs` in order to make `test` work, like so:

```console
nox -s docs
nox -s test
```

So then I modified `noxfile.py` so that, internally, the `test` session function does the same stuff that the `docs` session does.

You address a really good point, though, which is that I haven't really thought through the developer experience here, meaning, if I make a change to a component in the kitchen sink, and want to check that my changes pass the accessibility checker, what do I need to do? Part of the problem is that I don't really understand how the source code in this repo gets built. Specifically, I don't know what the relationship is between the `nox -s docs` command and the `nox -s compile` command.

Reply:
the compile command is creating the compiled JS and css assets from the sources. It is permformed at installation stage the first time. I have worked a lot in these files and nox -s compile allows me to run the coding cycle: 
- change assets
- compile
- build the docs 
- ... 
If I drop the compile stage, it continues to use the previously compiled asset which are deprecated. 
In my opinion it should stay, even though it's only useful for people working in the assets

Reply:
Ah thanks for that explanation. To re-iterate, I did **not** remove `nox -s compile`. I think you were looking at an inter-diff, which made it look like I had removed "`nox -s compile`" from the docs.